### PR TITLE
pixels as alias for dots where appr.

### PR DESCRIPTION
**Pull request by mdboom**

### CODE REVIEW

**Comment by efiring** [comment]
Now that I think about it, I'm not so sure I understand your chagrin at the "dots" usage.  After all, these are the same dots as in DPI.

**Comment by mdboom** [comment]
Yeah -- I see that now, too.  It's just not a word that gets used in Comp Graphics literature very much outside of "DPI".

File: lib/matplotlib/collections.py [hunk 1]
```diff
@@ -1507,8 +1507,8 @@ def __init__(self, widths, heights, angles, units='points', **kwargs):
         *angles*: sequence
             angles of first axes, degrees CCW from the X-axis
 
-        *units*: ['points' | 'inches' | 'dots' | 'width' | 'height'
```
Comment (tacaswell) [inline lib/matplotlib/collections.py]: This seems to piss sphinx off, maybe use `\`?  (or convert to numpdoc).

**Comment by mdboom** [comment]
I'm +0 on this, really, on further consideration.  Feel free to close.

**Comment by efiring** [comment]
OK, I will close this on the grounds that adding the alias in this case is adding complexity and the opportunity for confusion, with little if any compensating gain.

**Pull request closed without merging**